### ISSUE TITLE
Keep Control Focus on the Search Box

### ISSUE THREAD

**Issue by Davoleo** [YellowFish085/al-search#10]
Since the main feature of this extension is to look up entries on anilist it'd be cool
if the searchbox was focused automatically when opening the extension from the url bar.

I've also noticed that right after finishing to type the search query the search box is removed from control focus, I think this kinda worsens the overall extension UX, because if someone makes a typo and notices that after having finished to write they have to click the search bar again to edit their query string, a solution to this could be to keep the search bar focused always until the user clicks on one of the anime entries below or some other extension control

**Comment by YellowFish085** [comment]
Hello and thanks for the input.

Regarding the auto-focus when the popup is opened, I already tried to do it in the past (because it would obviously greatly improve the UX) but couldn't make it work. I also stumbled across multiple posts here and there about issues with auto-focus on add-ons popups, and I ultimately gave up on that. _But_, I've just quickly tested another option I apparently did not try and it appears that I can make it work for both Chrome and Firefox, so I'll update that soon.

For the second issue you mention, the input does get the focus back once the search results have loaded on Firefox, but I've realized that the behavior is not the same on Chrome. I'll find a fix for the Chrome version 👍 

Cheers
